### DISCUSSION TITLE
Is MBR the same as library refinement on the Cell Systems (2020) paper?

### DISCUSSION THREAD

**Discussion by luizalmeida93** [vdemichev/DiaNN#515] [Q&A] [answered]
Hi,

I started using DIA-NN this week, and I am exploring a few different settings. I read in the [Cell Systems (2020)](https://doi.org/10.1016/j.cels.2020.05.012) paper that it was performed a library refinement by re-running the data on the library generated by DIA-NN. After reading Match Between Runs (MBR) description, it appears to be the same thing, so I just wanted to double-check if I still need to re-run my data or if the MBR is already enough. Any thoughts?

Thank you in advance!

**Comment by vdemichev** [comment, marked as answer]
It is, but there's also special FDR control in addition to library refinement, it's not yet published.
